addons/base_automation/models/ir_actions_server.py

```diff
@@ -31,3 +33,50 @@ def _compute_available_model_ids(self):
         for action in rule_based:
             rule_model = action.base_automation_id.model_id
             action.available_model_ids = rule_model.ids if rule_model in action.available_model_ids else []
+
```

Comment:
the commit message doesn't match. Base automation stays in community, so it is not moved in studio

Reply:
same for the counterpart commit that removes that feature from mail

Reply:
indeed